test/e2e/features/cat-mod2/02-fail-journey.feature

```diff
@@ -2,16 +2,15 @@
 
 Feature: Driver Examiner complete the fail journey for Mod2
 
-  Background:
+  Scenario: Examiner fail the candidate with 11 driving faults
```

Comment:
The scenario name doesnt match up with the given, when, then scenario

Reply:
fixed